avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -76,6 +76,13 @@ param backendPoolType string = 'NodeIPConfiguration'
 ])
 param outboundType string = 'loadBalancer'
 
+@description('Optional. Name of a managed cluster SKU.')
```

Comment:
@ilhaan @JPEasier could you please review this contribution and help drive the PR to closure 💪 ?

Hope you don't mind if I chime in on one point. 
@zedy-wj since this PR is adding new input parameters, could you please update the minor version of the module as per semver? Thanks!

Reply:
Updated.